src/steam_network/protocol/protobuf_client.py

```diff
@@ -14,19 +13,73 @@
 import vdf
 
 from websockets.client import WebSocketClientProtocol
-from pprint import pformat
 
 from .consts import EMsg, EResult, EAccountType, EFriendRelationship, EPersonaState
-from .messages import steammessages_base_pb2, steammessages_clientserver_login_pb2, steammessages_auth_pb2, \
-    steammessages_player_pb2, steammessages_clientserver_friends_pb2, steammessages_clientserver_pb2, \
-    steammessages_chat_pb2, steammessages_clientserver_2_pb2, steammessages_clientserver_userstats_pb2, \
-    steammessages_clientserver_appinfo_pb2, resolved_service_messages_pb2, \
-    enums_pb2
+from .messages.steammessages_base_pb2 import (
+    CMsgMulti,
+    CMsgProtoBufHeader,
+)
+from .messages.steammessages_clientserver_login_pb2 import (
+    CMsgClientAccountInfo,
+    CMsgClientHeartBeat,
+    CMsgClientHello,
+    CMsgClientLoggedOff,
+    CMsgClientLogOff,
+    CMsgClientLogon,
+    CMsgClientLogonResponse,
+)
+from .messages.steammessages_auth_pb2 import (
+    CAuthentication_BeginAuthSessionViaCredentials_Request,
+    CAuthentication_BeginAuthSessionViaCredentials_Response,
```

Comment:
importing the class made it easier to get the messages for whatever new call we were making or adding type hints to the existing code, but this is definitely more python-3 friendly. If we're done with development and adding type hints, this is probably a good idea.

Reply:
I mostly did this to make the rest of the file more readable by removing the extremely long module prefix from type hints etc. Also, this way one can see at a glance which messages we need in case there is a change in the proto files again.